README.md

```diff
@@ -1,133 +1,111 @@
+### Current Features
 
- - Connect to Ankermake M5 and Ankermake APIs without using closed-source Anker
-   software.
+ - Print directly from PrusaSlicer and its derivatives (SuperSlicer, Bamboo Studio, OrcaSlicer, etc.)
 
- - Send raw gcode commands directly to the printer (and see the response)
+ - Connect to AnkerMake M5 and AnkerMake APIs without using closed-source Anker software.
 
- - Low-level access to MQTT, PPPP and HTTPS APIs
+ - Send raw gcode commands directly to the printer (and see the response).
 
- - Send print jobs to the printer
+ - Low-level access to MQTT, PPPP and HTTPS APIs.
 
- - Stream camera image/video to your computer
+ - Send print jobs (gcode files) to the printer.
 
+ - Stream camera image/video to your computer.
 
-## Upcoming and planned features
+ - Easily monitor print status.
 
- - Easily monitor print status
+### Upcoming and Planned Features
 
- - Integration into other software. Home Assistant? Prusa Slicer?
+ - Integration into other software. Home Assistant? Cura plugin?
 
-Pull requests always welcome :-)
+Let us know what you want to see; Pull requests always welcome! :smile:
 
-## Installation instructions
+## Installation (Prerequisite to the Usage section)
 
-First, please follow the [installation
-instructions](documentation/example-file-usage/example-file-prerequistes.md) for
-your platform.
+Follow the instructions for a [docker install](./documentation/install-from-docker.md) (recommended) or a [git install](./documentation/install-from-git.md).
 
-Verify that you can start `ankerctl.py`, and get the help screen:
+## Usage
 
-### For Windows, use: 
-```powershell
-python3 ankerctl.py -h
-```
-If that does not work then try running using `python ankerctl.py -h`
+### Web Interface
 
-### Linux and MacOS, use:
-```sh
-./ankerctl.py -h
-```
+1. Start the webserver by running one of the following commands in the folder you placed ankerctl in. You’ll need to have this running whenever you want to use the web interface or send jobs to the printer via a slicer:
 
-You should see the below output - if not then head back to [installation instructions](documentation/example-file-usage/example-file-prerequistes.md)
-```
-Usage: ankerctl.py [OPTIONS] COMMAND [ARGS]...
-
-Options:
-  -k, --insecure  Disable TLS certificate validation
-  -v, --verbose   Increase verbosity
-  -q, --quiet     Decrease verbosity
-  -h, --help      Show this message and exit.
-
-Commands:
-  config  View and update configuration
-  http    Low-level http api access
-  mqtt    Low-level mqtt api access
-  pppp    Low-level pppp api access
-```
+   docker:
 
-Before you can use ankerctl, you need to import the configuration.
+   ```sh
+   docker compose up
+   ```
 
-Support for logging in with username and password is not yet supported, but an
-`auth_token` can be imported from the saved credentials found in `login.json` in
-Ankermake Slicer. See `ankerctl.py config import -h` for details:
+   python:
```

Comment:
Should we call this "install from git" or something similar? either way, as long as it's consistent

Reply:
cleaned up with specific and consistent language.